description.md

```diff
@@ -21,6 +21,41 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Tutorials
 
+Tutorials are pieces of content directed toward the reproduction or verification of a practical result.
+Tutorials are step-by-step presentations of a topic, and are supposed to be read by participants individually.
```

Comment:
```suggestion
Tutorials are step-by-step presentations of a topic, and are supposed to be carried out by participants individually.
```
or "read and performed", so as to say they're not only about reading.

Reply:
From our experience, tutorials should focus on the steps and not on their results to incentivize students to run examples themselves.

Reply:
@Alex-deVis , you're right, I will rephrase accordingly.